src/redux/dragons/dragons.js

```diff
@@ -1,6 +1,30 @@
+import axios from 'axios';
+
+const DRAGONS_FETCHED = 'DRAGONS_FETCHED';
```

Comment:
**REQUIRED**
Please make sure to use the full namespace for the actions declared as per the [ducks](https://github.com/erikras/ducks-modular-redux) pattern.

Reply:
Thank you for your feedback @Danie12345 

I have made the required changed